srcjs/input_rate.js

```diff
@@ -298,30 +298,37 @@ var InputRateDecorator = function(target) {
 };
 (function() {
   this.setInput = function(name, value, opts) {
```

Comment:
As discussed in a meeting, this `name` should be renamed to `name_type` or `nameType` to make it clear that they accept strings like `"x:date"`, whereas other functions like `$doSetInput` only accept the name without type, as in `"x"`.